modules/rosa-sts-overview-of-the-default-cluster-specifications.adoc

```diff
@@ -62,6 +62,12 @@ ifdef::tf-classic,tf-hcp[]
 * Cluster name: `rosa-<6-digit-alphanumeric-string>`
 * Default AWS region for installations using the {cluster-manager-first} {hybrid-console-second}: us-east-2 (US East, Ohio)
 * Availability: Multi zone for the data plane
+* Default EC2 IMDS endpoint v2 is enabled
```

Comment:
Default EC2 IMDS endpoint v2 is enabled
what does this mean? default behavior is `optional`, does it means not enabled?

Reply:
This feature is not about whether v2 is enabled or not. v2 is always enabled. This feature is about enabling (making v2 optional) or disabling v1 (v2 required).

Reply:
I think it is important to clarify that this feature is not about whether IMDSv2 is enabled or not. Technically speaking, IMDSv2 is always "enabled" when the instance metadata service is enabled on an EC2 instance (OpenShift requires the instance metadata service to be enabled). Setting this feature to `optional` (which is the default) means that the EC2 instances can use IMDSv1 or IMDSv2 for instance metadata. In OpenShift's case, this means that the instances **will never use** IMDSv2, and will only use the less-secure IMDSv1. Setting this feature to `required` means that the EC2 instances use IMDSv2 for instance metadata and cannot use IMDSv1. This setting is enforced via an EC2 specific configuration which is managed by the Machine API. I hope this makes a little more sense.

Reply:
@bchandra-ocp I guess it would be nice to write "Default EC2 IMDS endpoint v1 and v2 is enabled"

Reply:
Done